src/dotnet/commands/dotnet-publish3/Program.cs

```diff
@@ -47,43 +45,17 @@ public static int Run(string[] args)
 
             app.OnExecute(() =>
             {
-                List<string> msbuildArgs = new List<string>();
+                Publish3Command publish = new Publish3Command();
 
-                if (!string.IsNullOrEmpty(projectArgument.Value))
-                {
-                    msbuildArgs.Add(projectArgument.Value);
-                }
+                publish.ProjectPath = projectArgument.Value;
+                publish.Framework = frameworkOption.Value();
+                publish.Runtime = runtimeOption.Value();
+                publish.OutputPath = outputOption.Value();
+                publish.Configuration = configurationOption.Value();
+                publish.VersionSuffix = versionSuffixOption.Value();
+                publish.ExtraMSBuildArguments = app.RemainingArguments;
```

Comment:
Should we add anywhere that you can pass additional parameters and that they will be forwarded to msbuild? We probably need to do that for all commands.